src/MoBi.R/Services/ProjectTask.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using MoBi.Core.Domain.Model;
+using MoBi.Core.Serialization.ORM;
+using OSPSuite.Core.Domain;
+
+namespace MoBi.R.Services
+{
+   public interface IProjectTask
+   {
+      MoBiProject GetProject(string fileName);
+      IReadOnlyList<string> GetModuleNames(MoBiProject moBiProject);
+      IReadOnlyList<string> GetSimulationNames(MoBiProject moBiProject);
+      IReadOnlyList<string> GetBuildingBlocksNamesFromModuleName(string moduleName);
+      IReadOnlyList<IMoBiSimulation> GetSimulations();
```

Comment:
I think this should return an `OSPSuite.R.Domain.Simulation`. That will give a consistent interface to R for simulations that are loaded using `OSPSuite.R.Services.SimulationPersistor`. It's just a friendly wrapper for R